tests/ha/single_pod_failure/test_server_pod_failure.py

```diff
@@ -1783,6 +1849,7 @@ def test_mpu_safe_server_pod_down(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-39915")
+    @pytest.mark.skip("Skipped until CORTX-33759 is fixed")
     @CTFailOn(error_handler)
     def test_mpu_unsafe_server_pod_down(self):
```

Comment:
shall we create new file for mpu, copy object and chunk test cases ? - test_server_pod_failure_api.py?

Reply:
Please ignore this comments as we decided we are not doing these change in single failure file